### DISCUSSION TITLE
Could someone pls how can I make MAS create the monika file without taking her out?

### DISCUSSION THREAD

**Discussion by CaniBeTheTweek** [Monika-After-Story/MonikaModDev#9139] [QA - code and content creation (technical)]
I want to make a custom roadtrip goodbye, but for that I need to find a way to do what the in-game version does aka what the "I'm going to take you somewhere" goodbye does just with different dialogue. Any type of help would be greatly appreciated :^

**Comment by JinSakaiM** [comment]
This could be more of a suggestion than a problem.

**Comment by ThePotatoGuy** [comment]
You could copy the existing farewell and make changes as desired.
